gulpfile.babel.js

```diff
@@ -91,7 +91,8 @@ watch.description = '`gulp watch` watches for changes and runs tasks automatical
 export const build = gulp.series(
 	clean,
 	handlebars,
-	gulp.parallel(processHtml, appTemplates, lint, fonts, images, clientScripts, vendorScripts, styles, bundleExternalCSS, copyStaticFiles, validateHtml, lintBootstrap, lintStyles, security, test),
+	gulp.parallel(processHtml, appTemplates, lint, fonts, images, clientScripts, vendorScripts, bundleExternalCSS, copyStaticFiles, validateHtml, lintBootstrap, lintStyles, security, test),
+	styles,
```

Comment:
I’m not that happy with this:
I don’t care that much if the production build takes one second more ore less, but this also affects the build time of the dev build. 

Compare dev build time from 10 runs below.

This branch:
```
[12:52:51] Finished 'build' after 9.09 s
[12:54:38] Finished 'build' after 8.51 s
[12:55:25] Finished 'build' after 8.65 s
[12:55:58] Finished 'build' after 8.09 s
[12:56:19] Finished 'build' after 7.77 s
[12:56:41] Finished 'build' after 7.89 s
[12:57:01] Finished 'build' after 8.16 s
[13:07:42] Finished 'build' after 7.98 s
[13:08:01] Finished 'build' after 7.96 s
[13:08:23] Finished 'build' after 7.94 s
```

vs. master branch:
```
[13:10:11] Finished 'build' after 7.86 s
[13:10:29] Finished 'build' after 7.82 s
[13:10:46] Finished 'build' after 7.67 s
[13:11:06] Finished 'build' after 7.75 s
[13:11:21] Finished 'build' after 7.93 s
[13:11:37] Finished 'build' after 8.2 s
[13:11:54] Finished 'build' after 8.06 s
[13:12:15] Finished 'build' after 7.99 s
[13:12:30] Finished 'build' after 7.8 s
[13:12:55] Finished 'build' after 7.96 s
```

On the other hand I don’t think it’s a good idea to set up the build task like that:

```javascript
/**
 * Define and export main build task:
 * Run `gulp build` respectively `gulp build --production`
 */
export let build;

if (isProdBuild()) {
	build = gulp.series(
		clean,
		handlebars,
		gulp.parallel(processHtml, appTemplates, lint, fonts, images, clientScripts, vendorScripts, bundleExternalCSS, copyStaticFiles, validateHtml, lintBootstrap, lintStyles, security, test),
		styles,
		cacheBust,
		banners
	);
} else {
	build = gulp.series(
		clean,
		handlebars,
		gulp.parallel(styles, processHtml, appTemplates, lint, fonts, images, clientScripts, vendorScripts, bundleExternalCSS, copyStaticFiles, validateHtml, lintBootstrap, lintStyles, security, test),
		cacheBust,
		banners
	);
}
build.description = '`gulp build` is the main build task';
build.flags = {
	'--production': ' builds for production to `dist` directory.',
	'-P': ' Alias for --production'
};
```

I don’t have a better idea right now.